src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -139,25 +139,23 @@ public void postPatronAccountByIdItemByItemIdRenew(String itemId, String id,
             return null;
           });
     } catch (Exception e) {
-      asyncResultHandler.handle(Future.succeededFuture(PostPatronAccountByIdItemByItemIdRenewResponse.withPlainInternalServerError(e.getMessage())));
+      asyncResultHandler.handle(Future.succeededFuture(PostPatronAccountItemRenewByIdAndItemIdResponse.respond500WithTextPlain(e.getMessage())));
       httpClient.closeClient();
     }
   }
 
   @Validate
   @Override
-  public void postPatronAccountByIdItemByItemIdHold(String itemId, String id,
+  public void postPatronAccountItemHoldByIdAndItemId(String id, String itemId,
       Hold entity, Map<String, String> okapiHeaders,
-      Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext)
-      throws Exception {
+      Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext) {
     final JsonObject holdJSON = new JsonObject()
         .put(JSON_FIELD_ITEM_ID, itemId)
```

Comment:
Agreed. I think I was only making them constants because of Sonar, which is why there is a mix now. There is still more mod-patron work, so we should keep this in mind.